packages/web/index.js

```diff
@@ -23,16 +25,30 @@ module.exports = (neutrino, opts = {}) => {
   const options = merge({
     env: [],
     hot: true,
-    html: {},
-    devServer: {
-      hot: opts.hot !== false
-    },
+    html: true,
```

Comment:
Can you explain the rationale for using 2 flags of `html` and `htmlTemplate`, instead of `html: false` vs. `html: {}`?

Reply:
Yeah…I think you're right, it isn't necessary.

To be clear: You're suggesting a single `html` setting where if object, it would pass options to @neutrinojs/html-template and if false, would do "the rest" (exclude htmlTemplate, script-ext, include manifest), yes?

Reply:
Yeah, I think that makes sense to me.